src/style/global.scss

```diff
@@ -2,6 +2,38 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import './index';
 
+html {
+  @include theme-variables(light);
+
+  @media (prefers-color-scheme: dark) {
+    @include theme-variables(dark);
+  }
+
+  @media (prefers-color-scheme: light) and (prefers-contrast: more) {
+    @include theme-variables(light-hc);
+  }
+
+  @media (prefers-color-scheme: dark) and (prefers-contrast: more) {
+    @include theme-variables(dark-hc);
+  }
+
+  &.iui-theme-light {
```

Comment:
Do you not need the extra & that was previously included with &.iui-theme-light? Looks like it was part of the fallback?

Reply:
I believe line 6 makes it so we don't need the extra `&`.

This:
```scss
html {
	@include theme-variables(light);

	&.iui-theme-light {
		@include theme-variables(light);
	}
}
```

Should have the same output as this:
```scss
html {
	&,
	&.iui-theme-light {
		@include theme-variables(light);
	}
}
```

The second option has less text, but I think the first option needs to be placed in this order so it runs through the media queries first & then if the class is applied, it overrides whatever was set by the media query.